mlc_llm/relax_model/commons.py

```diff
@@ -112,6 +131,7 @@ def moe_shard_gate_up_weight_scale(weight: relax.TensorStructInfo):
 
     return {
         "shard_qkv": shard_qkv_weight_scale,
+        "shard_qkv_bias": shard_bias,
         "shard_mlp_k": shard_k_weight_scale,
         "shard_o_proj_k": shard_k_weight_scale,
```

Comment:
I don't understand why the bias for output projection must not be sharded. Initially I sharded as well but the result was incorrect. Then I remember that the 1D scale for output projection in FT quantization must not be sharded as well https://github.com/mlc-ai/mlc-llm/blob/main/mlc_llm/relax_model/commons.py#L316-L320.

Reply:
if the shading is done for the reduction dimension, bias doesn't need to be shared, instead, bias need to be after all reduce and divided by num_shards